support/models.py

```diff
@@ -0,0 +1,55 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class FAQ(models.Model):
+    question = models.CharField(max_length=255)
+    answer = models.TextField()
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.question
+
+
+FAQ.objects.bulk_create([
```

Comment:
The call to bulk_create for the FAQ model is outside of any context (function or method), which causes an error because the ORM must be accessed in the Django runtime context (e.g., in a view function or Django shell).The call to bulk_create for the FAQ model is outside of any context (function or method), which causes an error because the ORM must be accessed in the Django runtime context (e.g., in a view function or Django shell).